TwitchPlaysAssembly/Src/Votes.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Timers;
+
+public static class Votes
+{
+	public static bool _going = false;
+	public static bool _global = false;
+	public static Dictionary<string, bool> Voters = new Dictionary<string, bool>();
+	public static TwitchBomb Bomb;
+
+	private static Timer Countdown = null;
+	private static string ActionType = "";
+
+	private static Dictionary<string, Action> actionDict = new Dictionary<string, Action>()
+	{
+		{"Detonation", () => Bomb.CauseExplosionByModuleCommand("Voting ended. Detonating bomb...", "Voted detonation")},
+		{"VS Mode Toggle", () => SendAndDo("Voting ended. Toggling VS Mode...", () => OtherModes.Toggle(TwitchPlaysMode.VS)) }
+	};
+	private static Dictionary<string, int> percentageDict = new Dictionary<string, int>()
+	{
+		{ "Detonation", TwitchPlaySettings.data.MinimumYesForDetonation },
+		{"VS Mode Toggle", TwitchPlaySettings.data.MinimumYesForVSMode }
+	};
```

Comment:
This whole dictionary should be in the settings file instead of having a setting for every type of vote.